ksql-engine/src/main/java/io/confluent/ksql/function/KsqlFunctionRegistry.java

```diff
@@ -46,12 +46,16 @@
 import java.util.List;
 import java.util.Map;
 
-public class KsqlFunctions {
+public class KsqlFunctionRegistry {
```

Comment:
Can rename this to FunctionRegistry  - we have an an explosion of KSQLEverything